packages/common-integration-tests/src/item-type.ts

```diff
@@ -0,0 +1,165 @@
+import {v4} from 'uuid';
+import {
+  ValidateCacheProps,
+  ItBehavesLikeItValidatesCacheName,
+  expectWithMessage,
+} from './common-int-test-utils';
+import {ICacheClient} from '@gomomento/sdk-core/dist/src/internal/clients/cache';
+import {ItemType} from '@gomomento/sdk-core';
+import {_ItemType} from '@gomomento/sdk-core/dist/src/messages/responses/grpc-response-types';
```

Comment:
What is the difference between these two imports?  `ItemType` vs `_ItemType`?  Are these both things that an end user might need to reference in their code?